src/docs/asciidoc/hazelcast_overview.adoc

```diff
@@ -293,6 +176,32 @@ In these cases, the partition table in the oldest member is updated with the new
 ownerships. Note that if a lite member joins or leaves a cluster, repartitioning is not triggered
 since lite members do not own any partitions.
 
+[[use-cases]]
+=== Use Cases
+
+Hazelcast can be used:
```

Comment:
I think we need to mention the CP subsystem here and in some other sections about capabilities as it's pretty awesome and interesting to some. Perhaps you can find inspiration in the intro sections of the CP subsystem docu?